packages/zoe/src/zoeService/types.js

```diff
@@ -112,12 +112,19 @@
  * @typedef {any} OfferResult
  */
 
+/**
+ * @typedef {Object} AdminFacet
+ * @property {() => Promise<string|Error|any>} getVatShutdownPromise
+ * @property {() => any} getVatStats
+ */
+
 /**
  * @typedef {Object} StartInstanceResult
  * @property {any} creatorFacet
  * @property {any} publicFacet
  * @property {Instance} instance
  * @property {Payment | undefined} creatorInvitation
+ * @property {AdminFacet} adminFacet
```

Comment:
Nice, I assume this had to be a different facet than creatorFacet because the creatorFacet is coming from the ZCF Vat but this object is coming from the Zoe Service Vat